packages/read-model/tests/Unit/Image/ImageViewRepositoryTest.php

```diff
@@ -0,0 +1,83 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Tests\ReadModelBundle\Unit\Image;
+
+use PHPUnit\Framework\TestCase;
+use Shopsys\FrameworkBundle\Component\Image\Image;
+use Shopsys\FrameworkBundle\Component\Image\ImageFacade;
+use Shopsys\ReadModelBundle\Image\ImageView;
+use Shopsys\ReadModelBundle\Image\ImageViewFactory;
+use Shopsys\ReadModelBundle\Image\ImageViewRepository;
+
+class ImageViewRepositoryTest extends TestCase
+{
+    private const IMAGE_EXTENSION = 'jpg';
+
+    /** @var \Shopsys\FrameworkBundle\Component\Image\ImageFacade */
+    private $imageFacadeMock;
+
+    protected function setUp()
+    {
+        parent::setUp();
+
+        $this->imageFacadeMock = $this->createMock(ImageFacade::class);
+        $this->imageFacadeMock->method('getImagesOrNullsByEntitiesIndexedByEntityId')
```

Comment:
This construction deserves an explanation... Please extract it into a function that will express what is the mock doing... maybe the whole mock creating could be in the function

Reply:
I hope it's better now

Reply:
nice